Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.4.2"
 [deps]
 InfrastructureModels = "2030c09a-7f63-5d83-885d-db604e0e9cc0"
 JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
+LightGraphs = "093fc24a-ae57-5d10-9952-331d41423f4d"
```

Comment:
Since LightGraphs.jl was archived back in October 2021, I think if we are going to add a new dependency it should probably be Graphs.jl. See here for more information https://github.com/JuliaGraphs/Graphs.jl#project-status

Reply:
Thanks David. Graphs.jl turned out to be a drop-in replacement.